test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -521,26 +521,13 @@ private static void MatchNumberWithTolerance(MatchCollection firstCollection, Ma
                 double f2 = double.Parse(secondCollection[i].ToString());
 
                 double allowedVariance = Math.Pow(10, -digitsOfPrecision);
-                double delta = Round(f1, digitsOfPrecision) - Round(f2, digitsOfPrecision);
+                double delta = Math.Round(f1, digitsOfPrecision) - Math.Round(f2, digitsOfPrecision);
```

Comment:
You should not be using `Math.Round`, it does not correctly handle significant digits that land on the left side of the decimal point (the integer part of the number). It only deals with the fractional half.

Reply:
re-introduced Round, thanks for the feedback. I moved to using it only on the difference, as rounding was not helping on a few cases. 

---
In reply to: [224629061](https://github.com/dotnet/machinelearning/pull/1206#discussion_r224629061) [](ancestors = 224629061)

Reply:
This would be a good comment in the code - why we have a `Round` method.

---
In reply to: [224671986](https://github.com/dotnet/machinelearning/pull/1206#discussion_r224671986) [](ancestors = 224671986,224629061)